src/screens/SendFunds/04-Summary.js

```diff
@@ -205,7 +238,7 @@ class SendSummary extends Component<
           <SummaryAmountSection
             account={account}
             parentAccount={parentAccount}
-            amount={amount}
+            amount={maxAmount || amount}
```

Comment:
isn't maxAmount always set? would it always display amount as a max in that code?